### ISSUE TITLE
Do accessibility research sessions for GOV.UK forms

### ISSUE THREAD

**Issue by Katrina-Birch** [alphagov/govuk-frontend#4409]
## What
Do accessibility research sessions for GOV.UK forms

## Who needs to work on this
User researcher

## Done when

- [ ] research sessions completed w/c 6/11

**Comment by kellylee-gds** [comment]
@Katrina-Birch we discussed this in sprint planning. Is this part of the v5 pre-release testing?

**Comment by Katrina-Birch** [comment]
No, this is doing research sessions with and for the Forms team with Oliver
Quinlan UR. I'm doing two sessions on 7/11 and one on 10/11

*Katrina Birch*
User Researcher
GOV.UK Design System <https://design-system.service.gov.uk/>
Government Digital Service
<https://www.gov.uk/government/organisations/government-digital-service>

*07810654511*
***@***.***
***@***.***>

City Tower, Piccadilly Plaza, Manchester, M1 4BT
The White Chapel Building, 10 Whitechapel High St, London E1 8QS
